docs/topics/secured-mission-intro.adoc

```diff
@@ -3,14 +3,18 @@
 
 The SSO booster project contains:
 
-* the sources for the Greeting service, which is the one which we are going to to secure
+* the sources for the Greeting service, which we are going to to secure
 * a template file (`service.sso.yaml`) to stand up the SSO server
 * the Keycloak adapter configuration to secure the service
 
 = Standing up {RHSSO}
-The `service.sso.yaml` file contains all OpenShift configuration items to stand up a pre-configured {RHSSO} server. The SSO server configuration has been simplified for the sake of this exercise and does provide an out-of-the-box configuration, with pre-configured users and security settings.
+The `service.sso.yaml` file contains:
 
-IMPORTANT: It is not recommended to use this SSO configuration in production. Specifically, the simplifications made to the booster security configuration impact the ability to use it in a production environment.
+* all OpenShift configuration items needed to stand up a pre-configured {RHSSO} server
```

Comment:
I am _really_ not sure about the use of _stand up_ here. Is this official RHSSO or OpenShift terminology? I coulnd't find much about this term on the internet, so please verify this is indeed the formulation we want to use.

Reply:
AFAIK,  _stand up_ is not standard OpenShift terminology and is often used as a substitute for _set up_ or _deploy_, I'd recommend using _deploy- instead of _stand up_, as ist appears the most common way used to refer to that procedure in the OpenShift and SSO documentation.

Reply:
Yea agree, I've never seen the word "stand up" used in this context before. Some people from some RHOAR teams tend to use it a lot, but I've personally never heard it used like this.

Reply:
I saw it in some non-RHOAR OpenShift-related communication, but that was very informal.